deployment.rb

```diff
@@ -91,6 +91,7 @@ def load_configuration()
     'dynamo_tables_table'         => "#{rack_env}_tables",
     'dynamo_tables_index'         => "channel_id-table_name-index",
     'dynamo_properties_table'     => "#{rack_env}_properties",
+    'throttle_data_apis'          => rack_env == :production,
```

Comment:
Any reason not to do this in non-production environments as well? Benefit would be that they then act more similarly to prod.